src/apps/pillarx-app/api/hey.json

```diff
@@ -0,0 +1,129 @@
+                  "balance": 587.64757917,
+                  "balanceRaw": "587647579169999945728",
+                  "chainId": "42161",
+                  "address": "0x912ce59144191c1204e64559fe8253a0e49e6548"
+                }
+              },
+              "contracts_balances": [
+                {
+                  "balance": 587.64757917,
+                  "balanceRaw": "587647579169999945728",
+                  "chainId": "42161",
+                  "address": "0x912ce59144191c1204e64559fe8253a0e49e6548",
+                  "decimals": 18
+                }
+              ]
+            },
+      
+            {
+              "asset": {
+                "name": "Orbit Protocol",
+                "symbol": "ORBIT",
+                "id": 102480544,
+                "contracts": [
+                  "0x42e12d42b3d6c4a74a88a61063856756ea2db357"
+                ],
+                "logo": "https://metacore.mobula.io/958c9264cc15c3fdf2d6ca544010be9d8385d373d7e441c6a9239e07af301e8c.png",
+                "blockchains": [
+                  "Blast"
+                ]
+              },
+              "realized_pnl": 0,
+              "unrealized_pnl": -29.3324,
+              "allocation": 0.15,
+              "price": 0.1383163581556911,
+              "price_bought": 0,
+              "price_change_24h": -14.07113,
+              "price_change_1h": 0,
+              "total_invested": 0,
+              "min_buy_price": 0.8745584448725503,
+              "max_buy_price": 0.8745584448725503,
+              "estimated_balance": 5.510617266881962,
+              "token_balance": 39.8406764056,
+              "cross_chain_balances": {
+                "Blast": {
+                  "balance": 39.84067640565484,
+                  "balanceRaw": "39840676405654839296",
+                  "chainId": "81457",
+                  "address": "0x42e12d42b3d6c4a74a88a61063856756ea2db357"
+                }
+              },
+              "contracts_balances": [
+                {
+                  "balance": 39.84067640565484,
+                  "balanceRaw": "39840676405654839296",
+                  "chainId": "81457",
+                  "address": "0x42e12d42b3d6c4a74a88a61063856756ea2db357",
+                  "decimals": 18
+                }
+              ]
+            }
+          ],
+          "total_pnl_history": {
+            "24h": {
+              "realized": 0,
+              "unrealized": -114.0661999999999
+            },
+            "7d": {
+              "realized": 0,
+              "unrealized": -169.20839999999998
+            },
+            "30d": {
+              "realized": 74.44840000000019,
+              "unrealized": 91.57700000000004
+            },
+            "1y": {
+              "realized": 350.1728000000002,
+              "unrealized": 955.3388000000001
+            }
+          }
+        },
+        "layout": "LIST_HORIZONTAL",
+        "id": "walletPortfolio"
+      }
+    ]
+  }
```

Comment:
Just a line ending note here

Reply:
Oooops, I was supposed to delete that file!